/jds/bin/diff -uprN libgnome-2.27.5.old/schemas/desktop_gnome_interface.schemas.in.in libgnome-2.27.5/schemas/desktop_gnome_interface.schemas.in.in
--- libgnome-2.27.5.old/schemas/desktop_gnome_interface.schemas.in.in	2009-09-08 15:36:19.832836656 +0100
+++ libgnome-2.27.5/schemas/desktop_gnome_interface.schemas.in.in	2009-09-08 15:38:25.997809931 +0100
@@ -65,7 +65,7 @@
       <applyto>/desktop/gnome/interface/menus_have_icons</applyto>
       <owner>gnome</owner>
       <type>bool</type>
-      <default>false</default>
+      <default>true</default>
       <locale name="C">
         <short>Menus Have Icons</short>
         <long>Whether menus may display an icon next to a menu entry.</long>
